README.md

```diff
@@ -1,44 +1,69 @@
-## scratch-www
+# scratch-www
```

Comment:
i changed all header levels to be the highest level available, except for the "Standalone web client for Scratch" subtitle below